### PR TITLE
Added concurrency to download loop

### PR DESCRIPTION
**Pull request by fervagar**
By using the 'deco' library, the downloading loop can be paralleled with only a couple of lines, to speed up the total downloading time without adding unnecessary complexity to the code.

### CODE REVIEW

**Comment by chaosAD** [comment]
Similar idea was earlier proposed in PR #66. I gave my opinion and some comment [there](https://github.com/alexgand/springer_free_books/pull/66#issuecomment-621688170). 

Springer host did not seem to play nice when it was overloaded (by the same source). So the chances of the downloaded files being incomplete are high. One main issue (not mentioned there) is that the Python script has no way to verify a downloaded file is good or corrupted, unless you manually do it.

In short, it was rejected by the owner (alexgand) of this project.

**Comment by fervagar** [comment]
Oh, I didn't see the previous PR, sorry! OK, no problem, it was just a proposal, but I understand both the technical and ethical issues regarding Springer's server overhead.

Greetings
